### ISSUE TITLE
Screen Reader cannot be deactivated

### ISSUE THREAD

**Issue by manuel-plavsic** [pop-os/cosmic-epoch#2518]
**Describe the bug**
After upgrading from beta.5 to beta.6, the system has the screen reader enabled and there is no way to stop it. Even if I toggle the "screen reader" setting, or reboot the system, the screen reader stays enabled. I absolutely need a workaround for the time being (is there some config file I can change manually?).

**To Reproduce**
1. Ensure the version running is the beta.6.
2. After logging in, the screen reader just starts (some of the actions that trigger the screen reader: increasing the volume with the keyboard, or opening a new Firefox tab with the mouse).

**Expected behavior**
The screen reader should respect the system settings.

**Screenshots**
No screenshots needed.

**Desktop (please complete the following information):**
 - OS: NixOS (unstable)

**Additional context**
- No additional context

**Comment by sachinchaudhary1808** [comment]
@manuel-plavsic hey i was trying kde too today, on the latest unstable version and it also happened on kde too, and on cosmic

so not sure it's specific cosmic issue, u can try `services.orca.enable = false;` it will disable that and it will still work with cosmic by enabling it in accessibility

**Comment by sachinchaudhary1808** [comment]
i noticed it first on kde

**Comment by manuel-plavsic** [comment]
@sachinchaudhary1808 great to know. Since it happened immediately after the upgrade of the beta version, I just attributed it to Cosmic. However, you experiencing the same also on KDE means it might very well be a NixOS bug. I'll leave it open for the moment, until what is causing this is clarified.

`services.orca.enable = false;` is the idiomatic way I didn't think of, thanks for suggesting it ;)

**Comment by me-it-is** [comment]
Has anyone made a bug report on nixpkgs

**Comment by manuel-plavsic** [comment]
Yes, it is here: https://github.com/NixOS/nixpkgs/issues/462935

A pull request addressing this was just merged. I am closing here.
